src/tests/conformance/post_handshake/validators.rs

```diff
@@ -18,7 +19,7 @@ const ONE_YEAR: u32 = 86400 * 365;
 const JAN1_2000: u32 = 946684800;
 const RAND_SEQUENCE_NUMBER: u32 = 2022102584;
 const MANIFEST_PREFIX: &[u8] = b"MAN\x00";
-const TIMEOUT_MILLIS: u128 = 5000;
+const TIMEOUT_MILLIS: Duration = Duration::from_secs(7);
```

Comment:
MILLIS? 🕵️‍♀️

Let's be more descriptive with file-scoped constants:
My suggestion, but feel free to change it:
```suggestion
const WAIT_MSG_TIMEOUT: Duration = Duration::from_secs(7);
```

Reply:
done